catalog/spark/spark.bom

```diff
@@ -91,6 +96,13 @@ brooklyn.catalog:
         minDisk: 100gb
         osFamily: centos
         osVersionRegex: 7
+        customizer:
+          $brooklyn:object:
+            type: org.apache.brooklyn.location.jclouds.networking.SharedLocationSecurityGroupCustomizer
+            object.fields:
+              tcpPortRanges:
+                - "4040-4050"
+              enabled: $brooklyn:config("spark.sharedsecuritygroup.create")
```

Comment:
This is better than forcing downstream users to include the logic as I did in https://github.com/brooklyncentral/brooklyn-spark/commit/e98d0cbc4dd952a8472df196c314c5ccaf32bf5c#diff-e3b8a2c17f156116494d47c3f7ca3c4dL98. Is it worth adding all the ports back or is `4040-4050` sufficient?